src/providers/flagsView.ts

```diff
@@ -133,7 +135,7 @@ export class LaunchDarklyTreeViewProvider implements vscode.TreeDataProvider<Fla
 			);
 		}
 
-		if (flag.tags) {
+		if (flag.tags.length > 0) {
 			let tags: Array<FlagValue> = [];
 			for (let i = 0; i < flag.tags.length; i++) {
```

Comment:
I typically prefer functional loops using `forEach` / `map` / `reduce` when applicable, and I think it would be a bit more succinct here. I'll leave this up to your style preference, though :)